src/test_l1_dsHdmiIn.c

```diff
@@ -278,19 +278,16 @@ void test_l1_dsHdmiIn_negative_dsHdmiInTerm(void) {
 void test_l1_dsHdmiIn_positive_dsHdmiInGetNumberOfInputs(void) {
     gTestID = 5;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID); 
-    uint8_t numInputs1, numInputs2;
```

Comment:
Line no. 272: Modify the descriptions and steps as we are now comparing with the profile file.

Reply:
Done